assured-workloads-example/main.tf

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  default_suffix             = random_string.id.result
+  aw_folder_name             = "${var.aw_name}-${local.default_suffix}"
+  encryption_keys_project_id = "${var.aw_base_id}-kms-${local.default_suffix}"
+  keyring_id                 = "${var.aw_base_id}-keyring-${local.default_suffix}"
+
+  aw_consumer_folder_id = one(
+    [for resource in google_assured_workloads_workload.primary.resources : resource.resource_id
+    if resource.resource_type == "CONSUMER_FOLDER"]
+  )
+
+  current_allowed_restricted_services = data.google_folder_organization_policy.aw_policy_restrict_service_usage_current.list_policy[0].allow[0].values
+
+  new_allowed_restricted_services = [
```

Comment:
I'd suggest to make this list of restricted services as a "base" and then make it "appendable" by a customer's input var.  or else make the whole thing something the customer needs to decide on, with a default as this list.